footer.php

```diff
@@ -2,33 +2,45 @@
 /**
  * The template for displaying the footer
  *
- * Contains the closing of the #content div and all content after.
- *
- * @link https://developer.wordpress.org/themes/basics/template-files/#template-partials
+ * Contains the closing of the #content div and all conten
  *
  * @package Jungo
  */
 
 ?>
 
-	</div><!-- #content -->
-
-	<footer id="colophon" class="site-footer">
-		<div class="site-info">
-			<a href="<?php echo esc_url( __( 'https://wordpress.org/', 'jungo' ) ); ?>">
-				<?php
-				/* translators: %s: CMS name, i.e. WordPress. */
-				printf( esc_html__( 'Proudly powered by %s', 'jungo' ), 'WordPress' );
-				?>
-			</a>
-			<span class="sep"> | </span>
-				<?php
-				/* translators: 1: Theme name, 2: Theme author. */
-				printf( esc_html__( 'Theme: %1$s by %2$s.', 'jungo' ), 'jungo', '<a href="https://www.linkedin.com/in/milan-poznan-0a0369109/">Milan Poznan</a>' );
-				?>
-		</div><!-- .site-info -->
-	</footer><!-- #colophon -->
-</div><!-- #page -->
+<section  class="footer">
+<div class="footer__wrapper">
+	<div class="footer__adress">
+		<p>ADDRESS</p>
+		<p>Vlajkoviceva 6 </p>
+		<p>Belgrade 11000  </p>
+		<p>Serbia </p>
+	</div>
+	<div class="footer__email">
+		<p>EMAIL</p>
+		<p>info@jungodigital.com</p>
+	</div>
+	<div class="footer__phone">
+		<p>PHONE</p>
+		<p>+381-(0)11-xx-xx-xx </p>
+	</div>
+	<div class="footer__connect">
+		<div class="footer__connect__wrapper">
+		 <p>CONNECT</p>
+		 <img src="wp-content/themes/jungo/assets/img/XMLID_444_.svg" alt="facebook logo"/>
```

Comment:
For images we will use WordPress function get_template_directory_uri() . /assets... I will explain to you what it is about. 
But gj for HTML proud of you man!